src/extension.ts

```diff
@@ -42,8 +42,8 @@ import { HelmTemplateCompletionProvider } from './helm.completionProvider';
 import { Reporter } from './telemetry';
 import * as telemetry from './telemetry-helper';
 import * as extensionapi from './extension.api';
-import {dashboardKubernetes} from './components/kubectl/dashboard';
-import {portForwardKubernetes} from './components/kubectl/port-forward';
+import { dashboardKubernetes } from './components/kubectl/dashboard';
```

Comment:
Can you separate out the tidying commits into a separate PR please?  I realise we will always get the odd bit of unrelated tidying in PRs, but there seems to be quite a lot in this one, and it obscures the functional changes.  Thanks!

Reply:
To be clear, the tidying changes are on point and are very welcome!

Reply:
will knock it off, as said in my previous PR its my vscode settings that has caused autoformat